fonts.css

```diff
@@ -0,0 +1,22 @@
+@font-face {
+	font-family: 'MuseoSansCyrl';
+	font-display: swap;
+	src: url("../fonts/MuseoSansCyrl.eot");
+	src: local("O"), url("../fonts/MuseoSansCyrl.woff") format("woff"), 
+			url("../fonts/MuseoSansCyrl.ttf") format("truetype"), 
+			url("../fonts/MuseoSansCyrl.svg") format("svg");
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'PTSerif-Bold';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/baliproject/pullRequest?prid=5558708)